sdk/ai/azure-ai-projects/samples/connections/sample_connections.py

```diff
@@ -23,13 +23,13 @@
        in your AI Studio Hub page.
     3) MODEL_DEPLOYMENT_NAME - The model deployment name, as found in your AI Studio Project.
 """
+from typing import cast
 
 import os
 from azure.ai.projects import AIProjectClient
 from azure.ai.projects.models import ConnectionType, AuthenticationType
 from azure.identity import DefaultAzureCredential
-
-#from azure.identity import DefaultAzureCredential, get_bearer_token_provider
+from azure.core.credentials import TokenCredential
```

Comment:
Please put this import statement above the line that uses it.

Reply:
Fixed